packages/evals/src/attachListeners.ts

```diff
@@ -14,11 +14,16 @@ export async function attachListeners(mastra?: Mastra) {
       await mastra.storage.insert({
         tableName: MastraStorage.TABLE_EVALS,
         record: {
-          result: JSON.stringify(traceObject.result),
-          meta: JSON.stringify(traceObject.meta),
           input: traceObject.input,
           output: traceObject.output,
-          createdAt: new Date().toISOString(),
+          result: traceObject.result,
+          agent_name: traceObject.agentName,
+          metric_name: traceObject.metricName,
+          instructions: traceObject.instructions,
+          test_info: traceObject.testInfo || null,
```

Comment:
do you have to JSON.stringify this? (looks like an object)
```suggestion
          test_info: traceObject.testInfo ? JSON.stringify(traceObject.testInfo) : null,
```

Reply:
Updated test_info and result to stringify value before insertion